ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -862,10 +862,11 @@ let empty_pool_uefi_certificates =
 let update_livepatch_guidance =
   {
     description=
-      "Replace reboot_host_on_livepatch_failure in host.pending_guidances with \
-       reboot_host_on_kernel_livepatch_failure and \
+      "1. Replace reboot_host_on_livepatch_failure in host.pending_guidances \
+       with reboot_host_on_kernel_livepatch_failure and \
        reboot_host_on_xen_livepatch_failure in \
-       host.pending_guidances_recommended"
+       host.pending_guidances_recommended. 2. Move the rest guidances in \
+       host.pending_guidances into host.pending_guidances_recommended"
   ; version= (fun _ -> true)
```

Comment:
Careful here: we don't want to do this for _every_ DB upgrade, only when moving from the old schema to the new.

Reply:
Sorry, forgot that!